thumbor/engines/pil.py

```diff
@@ -128,6 +135,18 @@ def create_image(self, buffer):
 
         return img
 
+    def get_exif_copyright(self):
+        exifs = piexif.load(self.image.info.get("exif"))
+        copyright_exif = {}
+        if exifs is None or "0th" not in exifs or exifs["0th"].items() is None:
```

Comment:
I may be missing something here, but I tried and failed to think of (and reproduce) cases where dict.items() could be `None`. So I believe this line should be corrected:
```suggestion
        if exifs is None or "0th" not in exifs or exifs["0th"] is None:
```